transcom/mymove
# MB-5994 Orders and allowances viewer url change to locator/move code

## Description

> Adjust the routes of the Orders and Allowances viewers as follows:
> 
> moves/:moveCode/orders - loads orders editor
> moves/:moveCode/allowances - loads allowances editor
> 
> This follows the convention in the [URLs spreadsheet](https://docs.google.com/spreadsheets/d/1VWtagZWTcmgqfWaLdQlUHKBJJVtqBXlEJ_KuMMOSZMs/edit#gid=0)

## Reviewer Notes

Is there anything you would like reviewers to give additional scrutiny?
- Had to modify the MoveDetails.jsx page to `/moves/:moveCode/details` in order to get the move code. We still would need to refactor the page to use react-query but that can be done in another story.
- I also had to modify the `/mto` page to use `moveCode` since I passed back the `moveCode` from the queue page

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate
make server_run
make office_client_run
```

Test
1. Log in as TOO, locally
2. Click on any move
3. Notice the details page url should now be moveCode, `moves/:moveCode/details`
4. Click on `View & edit orders`
5. Notice the orders page url should now be moveCode, `moves/:moveCode/orders`
6. Go back to details page
7. Click on `Edit allowances`
8. Notice the allowances page url should now be moveCode, `moves/:moveCode/allowances`
9. Go back to details page
10. Click on `Move task order` tab, noticed the url `moves/:moveCode/mto`

## Code Review Verification Steps



## References

* [Jira story](https://dp3.atlassian.net/browse/MB-5994) for this change

## Screenshots

Move details
![image](https://user-images.githubusercontent.com/1522549/102935847-c9020e80-445b-11eb-8401-4159e1c30e9f.png)

Move orders viewer
![image](https://user-images.githubusercontent.com/1522549/102935879-d8815780-445b-11eb-83ca-33dc9055cb81.png)

Move allowances viewer
![image](https://user-images.githubusercontent.com/1522549/102935911-e59e4680-445b-11eb-97bd-484c629719af.png)
